activities/PHQ-8/phq8_context.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+    "@context": {
+        "@version": 1.1,
+        "phq8_1": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_1.jsonld",
+            "@type": "@id"
+        },
+        "phq8_2": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_2.jsonld",
+            "@type": "@id"
+        },
+        "phq8_3": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_3.jsonld",
+            "@type": "@id"
+        },
+        "phq8_4": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_4.jsonld",
+            "@type": "@id"
+        },
+        "phq8_5": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_5.jsonld",
+            "@type": "@id"
+        },
+        "phq8_6": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_6.jsonld",
+            "@type": "@id"
+        },
+        "phq8_7": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_7.jsonld",
+            "@type": "@id"
+        },
+        "phq8_8": {
+            "@id": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/items/phq9_8.jsonld",
+            "@type": "@id"
+        },
+        "phq8_total_score": {
```

Comment:
i don't think this can be the same as `phq_total_score`. whenever we use a formulation like this, we are essentially assigning the key all the properties of the id to which this key is linked.

Reply:
sorry, that was a mistake! resolved it.